README.md

```diff
@@ -1,222 +1,73 @@
-
-1. Create a new component folder in **/dist/component/{name-of-your-component}**
-
-2. In your new component folder create the following files:
-
-    _(For a good starting place, see the template files in **/templates**)_
-
-    ```
-    /dist/components/component-name
-    -- _component-name.scss
-    -- component-name.dust
-    -- component-name.json
-    -- readme.md
-    ```
-    Note that the `*.scss` file should be prefixed with an underscore to avoid creating an unnecessary css file.
-
-3. Next, we should write some visual tests for our component. Create another component folder, but this time in **/tests/visual/components**
-
-4. In your new component test folder, create an `index.html` file
-
-    _(For a good starting place, see the `index.html` template in **/templates**)_
-
-5. Now create a `component-name.scss` file
-
-    _(Notice that this file should NOT have underscore in the file name. This is because we do want the `*.css` and `*.css.map` files that Sass will create)_
-
-6. In your newly created `scss` file, import **Vellum** and your **New Component styles**
-
-    ```
-    @import 'bower_components/vellum/dist/vellum';
-    @import 'dist/components/component-name/component-name';
-    ```
-
-    If you did everything correct saving `component-name.scss` will create `component-name.css` and `component-name.css.map` files and the Test folder will have the following structure:
-
-    ```
-    /tests/visual/components/component-name
-    -- index.html
-    -- component-name.css
-    -- component-name.css.map
-    -- component-name.scss
-    ```
-
-7. Files descriptions:
-
-    Write your SCSS styles in the **/dist/components/_component-name.scss**
-
-    Create and test your markup in the **/tests/visual/index.html**
-
-    When markup is done create its dust template in the **/dist/components/component-name.dust**
-
-    Fill json file with dummy data for dust template in **/dist/components/component-name.json**
+## Previewing Utilities
 
-    Write instucrions how to use this component in **/dist/components/readme.md**
+1. If you have Sass < 3.4, run `gem update sass`. If you don't have Bower installed, run `npm install -g bower`.
 
-## Updating `gh-pages` Online Docs
+2. In your utilities directory, run `npm install && bower install`.
 
-We use the `gh-pages` branch to host Stencil's visual test and make them accessible via the interwebs. In order to maintain these docs, we use [git-subtree](https://github.com/git/git/tree/master/contrib/subtree) to keep the `gh-pages` tests in sync with `master`.
+3. Run `grunt serve` to run testing environment locally. It will automatically recompile your stylesheets when you change them.
 
-Before you update the docs, make sure that `master` is fully up to date with all the latest tests and documentation.
+4. Navigate to `localhost:3000`.
```

Comment:
This could probably be simplified to... "4. Navigate to `localhost:3000/tests/visual`" and remove the below 5th step.